### ISSUE TITLE
Without Composer

### ISSUE THREAD

**Issue by fredtempez** [fire015/flintstone#45]
Hi
How to use Flintstone without Composer ?
Thanks

**Comment by damijanc** [comment]
One way would be to do `composer dump-autoload -o` This will create an file that you can then use in the use statement.

Other option would be to autoload files yourself [spl-autoload-register](https://www.php.net/manual/en/function.spl-autoload-register.php)

Third option would be to include all the files in the code.

What is the reason for not using the composer?

**Comment by fredtempez** [comment]
The project (ZwiiCMS) don't use composer but an original loading system. I'd like to keep this tree folder without composer structure.

Obviously I can run Flintstone with composer but not without it.

I tried first and third option without success. 
first - what file should I find and how use it ?
third - I loaded the 5 php file, but I still get error, should I download files in sub-dirs ?

Sorry for all theses questions and thanks giving me a sample.
